src/ducks/sharing/components/ShareByEmail.jsx

```diff
@@ -34,7 +35,8 @@ class ShareByUrl extends React.Component {
   constructor (props) {
     super(props)
     this.state = {
-      email: ''
+      email: '',
+      sharingType: 'master-slave'
```

Comment:
I think it would be better to use constants for `master-slave` and `master-master`.